mappings/com/mojang/blaze3d/platform/GLX.mapping

```diff
@@ -0,0 +1,218 @@
+		ARG 0 srcFrom
+		ARG 1 dstFrom
+		ARG 2 srcTo
+		ARG 3 dstTo
+	METHOD method_31807 uniform3 (ILjava/nio/FloatBuffer;)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 loc
+		ARG 1 v
+	METHOD method_31808 uniform3 (ILjava/nio/IntBuffer;)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 loc
+		ARG 1 v
+	METHOD method_31809 uniformMatrix4 (IZLjava/nio/FloatBuffer;)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 uniform
+		ARG 2 buffer
+	METHOD method_31810 createProgram ()I
+		COMMENT Creates a program object
+	METHOD method_31811 useProgram (I)V
+		COMMENT Installs a program object as part of current rendering state
+		ARG 0 program
+	METHOD method_31812 getShaderInfoLog (II)Ljava/lang/String;
+		COMMENT Returns the information log for a shader object
+		ARG 0 shader
+		ARG 1 maxLength
+	METHOD method_31813 uniform4 (ILjava/nio/FloatBuffer;)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 loc
+		ARG 1 v
+	METHOD method_31814 uniform4 (ILjava/nio/IntBuffer;)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 loc
+		ARG 1 v
+	METHOD method_31815 genBuffers ()I
+		COMMENT Generates buffer object names
+	METHOD method_31816 deleteProgram (I)V
+		COMMENT Deletes a program object
+		ARG 0 program
+	METHOD method_31817 getProgramInfoLog (II)Ljava/lang/String;
+		COMMENT Returns the information log for a program object
+		ARG 0 program
+		ARG 1 maxLength
+	METHOD method_31818 supportsVbo ()Z
+		COMMENT Returns whether the current GL context supports Vertex Buffer Objects
+	METHOD method_31819 linkProgram (I)V
+		COMMENT Links a program object
+		ARG 0 program
+	METHOD method_31820 uniform1 (II)V
+		COMMENT Specifies the value of a uniform variable for the current program object
+		ARG 0 loc
+		ARG 1 v
+	METHOD method_31821 genFramebuffers ()I
+		COMMENT Generates framebuffer object names
+	METHOD method_31822 deleteBuffers (I)V
+		COMMENT Deletes named buffer objects
+	METHOD method_31823 bindBuffer (II)V
+		COMMENT Binds a named buffer object
+	METHOD method_31824 genRenderbuffers ()I
+		COMMENT Generates framebuffer object names
+	METHOD method_31825 deleteRenderbuffers (I)V
+		COMMENT Deletes renderbuffer objects
+	METHOD method_31826 bindFramebuffer (II)V
+		COMMENT Binds a framebuffer to a framebuffer target
+	METHOD method_31827 deleteFramebuffers (I)V
+		COMMENT Deletes renderbuffer objects
+	METHOD method_31828 bindRenderbuffer (II)V
+		COMMENT Binds a renderbuffer to a renderbuffer target
+	METHOD method_31829 supportsFbo ()Z
+		COMMENT Returns whether the current GL context supports  Framebuffer Objects
+	METHOD method_31830 checkFramebufferStatus (I)I
+		COMMENT Checks the completeness of a framebuffer
+	METHOD method_31831 getProcessor ()Ljava/lang/String;
+	METHOD method_31832 activeTexture (I)V
```

Comment:
activeTextureUntracked

Reply:
this dispatches a call to `glActiveTexture`
hence `activeTexture`

Reply:
![image](https://user-images.githubusercontent.com/62552372/105660345-ce39f980-5ef0-11eb-9487-e7d09551a2e3.png)

Reply:
are you high? ``setActiveTexture``

Reply:
nope
we follow this convention everywhere in glstatemanager and glx
remove gl prefix and lowercase the first letter

Reply:
u r hy